src/Game/Managers/PartyManager.cs

```diff
@@ -53,68 +53,139 @@ public void ParsePacket(Packet p)
                     add = true;
                     goto case 2;
                 case 2:
-                    byte count = p.ReadByte();
 
-                    if (count <= 1)
+                    if (Engine.Profile.Current.CustomBarsToggled == true)
                     {
-                        Leader = 0;
-                        Inviter = 0;
+                        byte count = p.ReadByte();
 
-                        for (int i = 0; i < PARTY_SIZE; i++)
+                        if (count <= 1)
                         {
-                            if (Members[i] == null || Members[i].Serial == 0)
-                                break;
+                            Leader = 0;
+                            Inviter = 0;
 
-                            HealthBarGump gump = Engine.UI.GetGump<HealthBarGump>(Members[i].Serial);
+                            for (int i = 0; i < PARTY_SIZE; i++)
+                            {
+                                if (Members[i] == null || Members[i].Serial == 0)
+                                    break;
+
+                                HealthBarGumpCustom gump = Engine.UI.GetGump<HealthBarGumpCustom>(Members[i].Serial);
```

Comment:
Here search for both gumps.

var gump = Engine.UI.Gumps.OfType<AnchorableGump>().FirstOrDefault(s => s is HealthBarGumpCustom || s is HealthBarGump);

Reply:
discussed.